velox/dwio/common/SelectiveColumnReader.h

```diff
@@ -151,10 +151,9 @@ class SelectiveColumnReader {
   /**
    * Read the next group of values into a RowVector.
    * @param numValues the number of values to read
-   * @param vector to read into
+   * @param result vector to read into
    */
-  virtual void
-  next(uint64_t /*numValues*/, VectorPtr& /*result*/, const Mutation*) {
+  virtual void next(uint64_t numValues, VectorPtr& result, const Mutation*) {
```

Comment:
Keep the `/**/` around unused variables, otherwise we get lint warnings

Reply:
Emm would `[[maybe_unsued]]` tag also ok here?

Reply:
It's ok but it's longer